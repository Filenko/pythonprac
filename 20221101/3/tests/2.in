f, g, h = Grange(1, 2, 104853), Grange(1, 2, 1043576), Grange(1, 2, 48577)
o, e = Grange(1, 2, 1), Grange(100, 10, 20)
print(f"{f=}, {g=}, {h=}")
print(f"{list(o)}({not(o)}), {list(e)}({not(e)})")
print(f"{len(f)=}, {len(g)=}, {len(h)=}")
print(f"{len(list(f))=}, {len(list(g))=}, {len(list(h))=}")
print(f"{f[0]=}, {h[24]=}")
a, b = f[3:32767], h[3:32767:2]
print(f"{a=}, {a[5]=}")
print(f"{b=}, {b[5]=}, {list(b)=}")
