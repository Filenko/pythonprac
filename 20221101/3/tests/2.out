f=grange(1, 2, 104853), g=grange(1, 2, 1043576), h=grange(1, 2, 48577)
[](True), [](True)
len(f)=17, len(g)=20, len(h)=16
len(list(f))=17, len(list(g))=20, len(list(h))=16
f[0]=1, h[24]=16777216
a=grange(3, 2, 32767), a[5]=96
b=grange(3, 4, 32767), b[5]=3072, list(b)=[3, 12, 48, 192, 768, 3072, 12288]
